llm_studio/python_configs/text_dpo_modeling_config.py

```diff
@@ -27,7 +27,6 @@ class ConfigDPODataset(ConfigNLPCausalLMDataset):
     # Always have full chat history.
     # Chosen/Rejected prompt are only at the end of a conversation.
     limit_chained_samples: bool = True
-    mask_prompt_labels: bool = True
```

Comment:
I'd still leave it there, as this this explicitly needs to be True for DPO. Even if we would change defaults in the Base Class, it should be overwritten here.